packages/pn-personagiuridica-webapp/src/redux/apikeys/__test__/reducers.test.ts

```diff
@@ -0,0 +1,123 @@
+import MockAdapter from 'axios-mock-adapter';
+
+import { publicKeys, virtualKeys } from '../../../__mocks__/ApiKeys.mock';
+import { mockAuthentication } from '../../../__mocks__/Auth.mock';
+import { apiClient } from '../../../api/apiClients';
+import { ApiKeyAlgorithm, IssuerStatus } from '../../../models/ApiKeys';
+import { store } from '../../store';
+import {
+  checkPublicKeyIssuer,
+  createPublicKey,
+  createVirtualApiKey,
+  getPublicKeys,
+  getVirtualApiKeys,
+} from '../actions';
+
+const initialState = {
+  loading: false,
+  publicKeys: {
+    items: [],
+    total: 0,
+  },
+  virtualKeys: {
+    items: [],
+    total: 0,
+  },
+  publicKey: null,
```

Comment:
for what purpose do we need publicKey, virtualKey and issuerStatus?

Reply:
Since we don’t have to store data between steps, this has become useless